app/assets/stylesheets/widgets/form.scss

```diff
@@ -11,90 +15,95 @@
       font-size: 11px;
   }
 
+  label {
+    display: block;
+    margin: 0px;
+  }
+
   .inputs {
-    display: table;
+    display: table-row;
     width: 100%;
-    border-spacing: 5px;
 
-    .label {
+    .input {
       display: table-cell;
       padding: 5px;
     }
-  }
 
-  label, .input {
-    display: block;
-    margin: 0px;
-  }
-
-  .input > span {
+    .input > span {
```

Comment:
Merge rule `.input > span` with rule on line 27<br>Selector should have depth of applicability no greater than 2, but was 4